tests/unit/client.py

```diff
@@ -0,0 +1,442 @@
+# Copyright 2013-2014 Massachusetts Open Cloud Contributors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the
+# License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an "AS
+# IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+# express or implied.  See the License for the specific language
+# governing permissions and limitations under the License.
+
+"""Unit tests for client library"""
+import haas
+from haas import model, api, deferred, server, config
+from haas.model import db
+from haas.test_common import *
```

Comment:
Don't do `import *`. I know it's used elsewhere, but we want to remove that. Let's not make it worse. See also #678.